src/handlers.js

```diff
@@ -90,20 +95,56 @@ class Handlers {
     this.mountGlobal();
 
     if (TOUCH_ENABLED) {
-      element.addEventListener('touchstart', this.handleTouchStart);
-      element.addEventListener('touchmove', this.handleTouchMove);
-      element.addEventListener('touchend', this.handleTouchEnd);
+      element.addEventListener(
```

Comment:
Is this backward compatible with older browsers (e.g. 2-3 years old versions)?

Reply:
Yes: https://caniuse.com/mdn-api_eventtarget_addeventlistener_options_parameter_options_capture_parameter

Reply:
Thx.